README.md

```diff
@@ -33,70 +33,72 @@ We will define a `set` and fill it up with the tokens which current query should
 key_tokens = set()
 ```
 
-Nearby tokens are important obviously because in a sentence (sequence of words), current word is highly dependent on few future & past tokens. This idea introduced the concept of `sliding attention`.
+Nearby tokens are important because in a sentence (sequence of words), the current word is highly dependent on neighboring future & past tokens. This intuition is the idea behind the concept of `sliding attention`.
 
 ```python
 # Let's update `set` with nearby tokens
 sliding_tokens = ["<something>", "....."]
 key_tokens.update(sliding_tokens)
 ```
 
-**Long range dependencies:** long range relationships needs to be captured for lots of tasks. Eg: `question-answering` where model needs to capture information about entire question and most of context to be able to answer correctly.
+**Long range dependencies:** For some tasks, it is crucial that the model is able to capture long range relationships between tokens. *E.g.*, in `question-answering` the model needs to compare each token of the context to the whole question to be able to figure out which part of the context is useful for a correct answer. If most of the context tokens would just attend to other context tokens, but not to the question, it becomes much harder for the model to filter important context tokens from less important context tokens.
 
-there are 2 ways of doing this:
+Now, `BigBird` proposes two ways of allowing long-term attention dependencies while staying computationally efficient.
 
-* Introduce some tokens which will attend every token and gets attented by all the tokens. Eg: "HuggingFace is building nice libraries for easy NLP". Now, let's say 'building' is global token, and we want to associate 'NLP' with 'HuggingFace' for some task; Now 'building' representation will possibly help model to assiciate 'NLP' with 'HuggingFace' since both tokens will get associated by 'building'.
+* Introduce some tokens which will attend to every token and which are attented by every token. Eg: "HuggingFace is building nice libraries for easy NLP". Now, let's say 'building' is defined as a global token, and we want to associate 'NLP' with 'HuggingFace' for some task; Now having 'building' attend globally to all other tokens will probably help the model to assiciate 'NLP' with 'HuggingFace'.
 
 ```python
 # fill up global tokens in our `set`
 global_tokens = ["<something>", "....."]
```

Comment:
Same here, compared to the code above this is hard to understand -> what is "<something>" ? How does this differ to sliding attention?